### PR TITLE
kibana 5.4.0

### PR DESCRIPTION
**Pull request by jasontedor**
This pull request updates the Kibana version from 5.3.2 to 5.4.0.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@jasontedor install seems to have gone :boom: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/811/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_kibana/

**Comment by ilovezfs** [comment]
@jasontedor I think kibana is failing due to https://github.com/angular-translate/grunt-angular-translate/issues/84

Should we pass `--force`?

File: Formula/kibana.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def install
     ENV.prepend_path "PATH", prefix/"libexec/node/bin"
     Pathname.new("#{ENV["HOME"]}/.npmrc").write Language::Node.npm_cache_config
     system "npm", "install", "--verbose"
-    system "npm", "run", "build", "--", "--release", "--skip-os-packages", "--skip-archives"
+    system "npm", "run", "build", "--", "--release", "--skip-os-packages", "--skip-archives", "--force"
```
Comment (jasontedor) [inline Formula/kibana.rb]: I haven't had time to dig into what is going on here, but I don't think we should do this, it's not needed when we build the release bits.

**Review by jasontedor — COMMENTED** [review]
I left a comment.

**Comment by ilovezfs** [comment]
The failure on Sierra looks like a parallelization problem.

File: Formula/kibana.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/kibana.rb]: @jasontedor OK, I reverted it

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@jasontedor Thanks for the pull request. Shipped! It looks like the build issue spontaneously resolved probably due to a fix in one of the node dependencies.

**Comment by jasontedor** [comment]
@ilovezfs Awesome, thank you!

**Comment by ilovezfs** [comment]
You're welcome!

**Pull request closed without merging**